sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 1.1.0-beta.4 (Unreleased)
+## 1.1.0-beta.4 (2020-05-06)
```

Comment:
Mind adding in the missing patch release history for 1.0.4, 1.0.5, and 1.0.6 below 1.1.0-beta.1

Reply:
Shouldn't this be ordered via the date they released ?

Reply:
By date would be best